File: guns-vip-main/src/main/webapp/assets/studentTotal/studentTotal.js
layui.use(['table', 'admin', 'ax', 'func'], function () {
    var $ = layui.$;
    var table = layui.table;
    var $ax = layui.ax;
    var admin = layui.admin;
    var func = layui.func;

    /**
     * 管理
     */
    var StudentTotal = {
        tableId: "studentTotalTable"
    };

    /**
     * 初始化表格的列
     */
    StudentTotal.initColumn = function () {
        return [[
            {type: 'checkbox'},
            {field: 'stuName', sort: true, title: '姓名'},
            {field: 'pstuName', sort: true, title: '曾用名'},
            // {field: 'photoSrc', sort: true, title: '个人照片',minWidth: 120},
            {field: 'stuNumber', sort: true, title: '学号'},
            {field: 'cardNumber', sort: true, title: '一卡通号',minWidth: 120},
            {field: 'idCard', sort: true, title: '证件号码',minWidth: 120},
            {field: 'phoneNumber', sort: true, title: '联系电话',minWidth: 120},
            {field: 'birth', sort: true, title: '出生日期',minWidth: 120},
            {field: 'gender', sort: true, title: '性别'},
            {field: 'nation', sort: true, title: '民族'},
            {field: 'bloodType', sort: true, title: '血型'},
            {field: 'householdRegisterType', sort: true, title: '户籍类别',minWidth: 120},
            {field: 'youthLeagueDate', sort: true, title: '入团日期',minWidth: 120},
            {field: 'youthLeagueNum', sort: true, title: '团员发展编号', minWidth: 140},
            {field: 'residenceAddr', sort: true, title: '户籍地址',minWidth: 120},
            {field: 'currentAddr', sort: true, title: '现居住地',minWidth: 120},
            // {field: 'householdRegisterFront', sort: true, title: '户口本首页', minWidth: 140},
            // {field: 'householdRegisterMain', sort: true, title: '户口本主页', minWidth: 140},
            // {field: 'householdRegisterSelf', sort: true, title: '户口本本人页', minWidth: 140},
            {field: 'stuType', sort: true, title: '学生类别',minWidth: 120},
            {field: 'stuClass', sort: true, title: '班级'},
            {field: 'schoolNum', sort: true, title: '省内学校代码', minWidth: 140},
            {field: 'totalScore', sort: true, title: '入学总分',minWidth: 120},
            {field: 'testScore', sort: true, title: '中考文化课成绩', minWidth: 140},
            {field: 'graduateSchool', sort: true, title: '原毕业院校', minWidth: 140},
            {field: 'registerType', sort: true, title: '学生来源',minWidth: 120},
            {field: 'studyType', sort: true, title: '入学方式',minWidth: 120},
            {field: 'admitOrder', sort: true, title: '中招序号',minWidth: 120},
            {field: 'followClass', sort: true, title: '随班就读',minWidth: 120},
            {field: 'accommodation', sort: true, title: '是否住宿生',minWidth: 140},
            {field: 'studyMode', sort: true, title: '就读方式',minWidth: 120},
            {field: 'migrantWorkersChildren', sort: true, title: '是否随迁子女', minWidth: 140},
            {field: 'countryWorkersChildren', sort: true, title: '是否进城务工随迁子女', minWidth: 200},
            {field: 'governmentPurchaseDegree', sort: true, title: '是否由政府购买学位', minWidth: 200},
            {field: 'enrollmentstatus', sort: true, title: '学籍变更备注', minWidth: 140},
            {field: 'politicalStatus', sort: true, title: '政治面貌',minWidth: 120},
            {field: 'hobby', sort: true, title: '特长'},
            {field: 'principalSheet', sort: true, title: '主页地址',minWidth: 120},
            {field: 'healthCondition', sort: true, title: '健康状况',minWidth: 120},
            {field: 'leftBehindChildren', sort: true, title: '是否为留守儿童', minWidth: 160},
            {field: 'disableType', sort: true, title: '残疾类型',minWidth: 120},
            {field: 'orphan', sort: true, title: '是否孤儿',minWidth: 120},
            {field: 'ruralDibao', sort: true, title: '是否农村低保', minWidth: 140},
            {field: 'extremelyPoor', sort: true, title: '是否农村特困救助供养', minWidth: 240},
            {field: 'singleParent', sort: true, title: '是否单亲',minWidth: 120},
            {field: 'solderChildren', sort: true, title: '是否军人子女', minWidth: 160},
            {field: 'causesOfPoverty', sort: true, title: '家庭是否有其他重大致贫因素', minWidth: 260},
            {field: 'applySponsor', sort: true, title: '是否需要申请资助', minWidth: 200},
            {field: 'needSupply', sort: true, title: '是否享受一补', minWidth: 140},
            {field: 'onlyChild', sort: true, title: '是否为独生子女', minWidth: 170},
            {field: 'martyrChildren', sort: true, title: '是否烈士或优抚子女',minWidth: 200},
            {field: 'healthfulDiet', sort: true, title: '是否享受营养餐', minWidth: 200},
            {field: 'putOnRecord', sort: true, title: '是否建档立卡贫困户在校生', minWidth: 250},
            // {field: 'deleteMember', sort: true, title: '删除'},
            {field: 'ordernumber', hide: true, title: ''},
            {align: 'center', toolbar: '#tableBar', title: '操作', minWidth: 200}
        ]];
    };

    /**
     * 点击查询按钮
     */
    StudentTotal.search = function () {
        var queryData = {};


        table.reload(StudentTotal.tableId, {
            where: queryData, page: {curr: 1}
        });
    };

    /**
     * 跳转到添加页面
     */
    StudentTotal.jumpAddPage = function () {
        window.location.href = Feng.ctxPath + '/studentTotal/add'
    };

    /**
     * 跳转到编辑页面
     *
     * @param data 点击按钮时候的行数据
     */
    StudentTotal.jumpEditPage = function (data) {
        window.location.href = Feng.ctxPath + '/studentTotal/edit?ordernumber=' + data.ordernumber
    };

    /**
     * 导出excel按钮
     */
    StudentTotal.exportExcel = function () {
        var checkRows = table.checkStatus(StudentTotal.tableId);
        if (checkRows.data.length === 0) {
            Feng.error("请选择要导出的数据");
        } else {
            table.exportFile(tableResult.config.id, checkRows.data, 'xls');
        }
    };

    /**
     * 点击删除
     *
     * @param data 点击按钮时候的行数据
     */
    StudentTotal.onDeleteItem = function (data) {
        var operation = function () {
            var ajax = new $ax(Feng.ctxPath + "/studentTotal/delete", function (data) {
                Feng.success("删除成功!");
                table.reload(StudentTotal.tableId);
            }, function (data) {
                Feng.error("删除失败!" + data.responseJSON.message + "!");
            });
            ajax.set("ordernumber", data.ordernumber);
            ajax.start();
        };
        Feng.confirm("是否删除?", operation);
    };

    // 渲染表格
    var tableResult = table.render({
        elem: '#' + StudentTotal.tableId,
        url: Feng.ctxPath + '/studentTotal/list',
        page: true,
        height: "full-158",
        cellMinWidth: 100,
        cols: StudentTotal.initColumn()
    });

    // 搜索按钮点击事件
    $('#btnSearch').click(function () {
        StudentTotal.search();
    });

    // 添加按钮点击事件
    $('#btnAdd').click(function () {

        StudentTotal.jumpAddPage();

    });

    // 导出excel
    $('#btnExp').click(function () {
        StudentTotal.exportExcel();
    });

    // 工具条点击事件
    table.on('tool(' + StudentTotal.tableId + ')', function (obj) {
        var data = obj.data;
        var layEvent = obj.event;

        if (layEvent === 'edit') {
            StudentTotal.jumpEditPage(data);
        } else if (layEvent === 'delete') {
            StudentTotal.onDeleteItem(data);
        }
    });
});

File: guns-vip-main/src/main/java/cn/stylefeng/guns/modular/students/entity/StudentTotal.java
package cn.stylefeng.guns.modular.students.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import com.baomidou.mybatisplus.annotation.IdType;
import java.util.Date;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableField;
import java.io.Serializable;

/**
 * <p>
 * 
 * </p>
 *
 * @author yl 
 * @since 2020-10-10
 */
@TableName("student_total")
public class StudentTotal implements Serializable {

    private static final long serialVersionUID=1L;

    /**
     * 姓名
     */
    @TableField("stu_name")
    private String stuName;

    /**
     * 曾用名
     */
    @TableField("pstu_name")
    private String pstuName;

    /**
     * 个人照片
     */
    @TableField("photo_src")
    private String photoSrc;

    /**
     * 学号
     */
    @TableField("stu_number")
    private Long stuNumber;

    /**
     * 一卡通号
     */
    @TableField("card_number")
    private Long cardNumber;

    /**
     * 证件号码
     */
    @TableField("id_card")
    private String idCard;

    /**
     * 联系电话
     */
    @TableField("phone_number")
    private Long phoneNumber;

    /**
     * 出生日期
     */
    @TableField("birth")
    private Date birth;

    /**
     * 性别
     */
    @TableField("gender")
    private Integer gender;

    /**
     * 民族
     */
    @TableField("nation")
    private Integer nation;

    /**
     * 血型
     */
    @TableField("blood_type")
    private Integer bloodType;

    /**
     * 户籍类别
     */
    @TableField("Household_Register_type")
    private Integer householdRegisterType;

    /**
     * 入团日期
     */
    @TableField("youth_league_date")
    private Date youthLeagueDate;

    /**
     * 团员发展编号
     */
    @TableField("youth_league_num")
    private String youthLeagueNum;

    /**
     * 户籍地址
     */
    @TableField("residence_addr")
    private String residenceAddr;

    /**
     * 现居住地
     */
    @TableField("current_addr")
    private String currentAddr;

    /**
     * 户口本首页
     */
    @TableField("Household_Register_front")
    private String householdRegisterFront;

    /**
     * 户口本主页
     */
    @TableField("Household_Register_main")
    private String householdRegisterMain;

    /**
     * 户口本本人页
     */
    @TableField("Household_Register_self")
    private String householdRegisterSelf;

    /**
     * 学生类别
     */
    @TableField("stu_type")
    private Integer stuType;

    /**
     * 班级
     */
    @TableField("stu_class")
    private String stuClass;

    /**
     * 省内学校代码
     */
    @TableField("school_num")
    private String schoolNum;

    /**
     * 入学总分
     */
    @TableField("total_score")
    private Integer totalScore;

    /**
     * 中考文化课成绩
     */
    @TableField("test_score")
    private Integer testScore;

    /**
     * 原毕业院校
     */
    @TableField("graduate_school")
    private String graduateSchool;

    /**
     * 学生来源
     */
    @TableField("register_type")
    private Integer registerType;

    /**
     * 入学方式
     */
    @TableField("study_type")
    private Integer studyType;

    /**
     * 中招序号
     */
    @TableField("admit_order")
    private Long admitOrder;

    /**
     * 随班就读
     */
    @TableField("follow_class")
    private Integer followClass;

    /**
     * 是否住宿生
     */
    @TableField("accommodation")
    private Integer accommodation;

    /**
     * 就读方式
     */
    @TableField("study_mode")
    private Integer studyMode;

    /**
     * 是否随迁子女
     */
    @TableField("Migrant_Workers_children")
    private Integer migrantWorkersChildren;

    /**
     * 是否进城务工随迁子女
     */
    @TableField("country_workers_children")
    private Integer countryWorkersChildren;

    /**
     * 是否由政府购买学位
     */
    @TableField("government_purchase_degree")
    private Integer governmentPurchaseDegree;

    /**
     * 学籍变更备注
     */
    @TableField("enrollmentstatus")
    private String enrollmentstatus;

    /**
     * 政治面貌
     */
    @TableField("Political_Status")
    private Integer politicalStatus;

    /**
     * 特长
     */
    @TableField("hobby")
    private String hobby;

    /**
     * 主页地址
     */
    @TableField("principal_sheet")
    private String principalSheet;

    /**
     * 健康状况
     */
    @TableField("health_condition")
    private Integer healthCondition;

    /**
     * 是否为留守儿童
     */
    @TableField("left_behind_children")
    private Integer leftBehindChildren;

    /**
     * 残疾类型
     */
    @TableField("disable_type")
    private Integer disableType;

    /**
     * 是否孤儿
     */
    @TableField("orphan")
    private Integer orphan;

    /**
     * 是否农村低保
     */
    @TableField("rural_dibao")
    private Integer ruralDibao;

    /**
     * 是否农村特困救助供养
     */
    @TableField("extremely_poor")
    private Integer extremelyPoor;

    /**
     * 是否单亲
     */
    @TableField("single_parent")
    private Integer singleParent;

    /**
     * 是否军人子女
     */
    @TableField("solder_children")
    private Integer solderChildren;

    /**
     * 家庭是否有其他重大致贫因素
     */
    @TableField("causes_of_poverty")
    private Integer causesOfPoverty;

    /**
     * 是否需要申请资助
     */
    @TableField("apply_sponsor")
    private Integer applySponsor;

    /**
     * 是否享受一补
     */
    @TableField("need_supply")
    private Integer needSupply;

    /**
     * 是否为独生子女
     */
    @TableField("only_child")
    private Integer onlyChild;

    /**
     * 是否烈士或优抚子女
     */
    @TableField("Martyr_children")
    private Integer martyrChildren;

    /**
     * 是否享受营养餐
     */
    @TableField("healthful_diet")
    private Integer healthfulDiet;

    /**
     * 是否建档立卡贫困户在校生
     */
    @TableField("put_on_record")
    private Integer putOnRecord;

    /**
     * 删除
     */
    @TableField("delete_member")
    private Integer deleteMember;

      @TableId(value = "ordernumber", type = IdType.ID_WORKER)
    private String ordernumber;


    public String getStuName() {
        return stuName;
    }

    public void setStuName(String stuName) {
        this.stuName = stuName;
    }

    public String getPstuName() {
        return pstuName;
    }

    public void setPstuName(String pstuName) {
        this.pstuName = pstuName;
    }

    public String getPhotoSrc() {
        return photoSrc;
    }

    public void setPhotoSrc(String photoSrc) {
        this.photoSrc = photoSrc;
    }

    public Long getStuNumber() {
        return stuNumber;
    }

    public void setStuNumber(Long stuNumber) {
        this.stuNumber = stuNumber;
    }

    public Long getCardNumber() {
        return cardNumber;
    }

    public void setCardNumber(Long cardNumber) {
        this.cardNumber = cardNumber;
    }

    public String getIdCard() {
        return idCard;
    }

    public void setIdCard(String idCard) {
        this.idCard = idCard;
    }

    public Long getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(Long phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public Date getBirth() {
        return birth;
    }

    public void setBirth(Date birth) {
        this.birth = birth;
    }

    public Integer getGender() {
        return gender;
    }

    public void setGender(Integer gender) {
        this.gender = gender;
    }

    public Integer getNation() {
        return nation;
    }

    public void setNation(Integer nation) {
        this.nation = nation;
    }

    public Integer getBloodType() {
        return bloodType;
    }

    public void setBloodType(Integer bloodType) {
        this.bloodType = bloodType;
    }

    public Integer getHouseholdRegisterType() {
        return householdRegisterType;
    }

    public void setHouseholdRegisterType(Integer householdRegisterType) {
        this.householdRegisterType = householdRegisterType;
    }

    public Date getYouthLeagueDate() {
        return youthLeagueDate;
    }

    public void setYouthLeagueDate(Date youthLeagueDate) {
        this.youthLeagueDate = youthLeagueDate;
    }

    public String getYouthLeagueNum() {
        return youthLeagueNum;
    }

    public void setYouthLeagueNum(String youthLeagueNum) {
        this.youthLeagueNum = youthLeagueNum;
    }

    public String getResidenceAddr() {
        return residenceAddr;
    }

    public void setResidenceAddr(String residenceAddr) {
        this.residenceAddr = residenceAddr;
    }

    public String getCurrentAddr() {
        return currentAddr;
    }

    public void setCurrentAddr(String currentAddr) {
        this.currentAddr = currentAddr;
    }

    public String getHouseholdRegisterFront() {
        return householdRegisterFront;
    }

    public void setHouseholdRegisterFront(String householdRegisterFront) {
        this.householdRegisterFront = householdRegisterFront;
    }

    public String getHouseholdRegisterMain() {
        return householdRegisterMain;
    }

    public void setHouseholdRegisterMain(String householdRegisterMain) {
        this.householdRegisterMain = householdRegisterMain;
    }

    public String getHouseholdRegisterSelf() {
        return householdRegisterSelf;
    }

    public void setHouseholdRegisterSelf(String householdRegisterSelf) {
        this.householdRegisterSelf = householdRegisterSelf;
    }

    public Integer getStuType() {
        return stuType;
    }

    public void setStuType(Integer stuType) {
        this.stuType = stuType;
    }

    public String getStuClass() {
        return stuClass;
    }

    public void setStuClass(String stuClass) {
        this.stuClass = stuClass;
    }

    public String getSchoolNum() {
        return schoolNum;
    }

    public void setSchoolNum(String schoolNum) {
        this.schoolNum = schoolNum;
    }

    public Integer getTotalScore() {
        return totalScore;
    }

    public void setTotalScore(Integer totalScore) {
        this.totalScore = totalScore;
    }

    public Integer getTestScore() {
        return testScore;
    }

    public void setTestScore(Integer testScore) {
        this.testScore = testScore;
    }

    public String getGraduateSchool() {
        return graduateSchool;
    }

    public void setGraduateSchool(String graduateSchool) {
        this.graduateSchool = graduateSchool;
    }

    public Integer getRegisterType() {
        return registerType;
    }

    public void setRegisterType(Integer registerType) {
        this.registerType = registerType;
    }

    public Integer getStudyType() {
        return studyType;
    }

    public void setStudyType(Integer studyType) {
        this.studyType = studyType;
    }

    public Long getAdmitOrder() {
        return admitOrder;
    }

    public void setAdmitOrder(Long admitOrder) {
        this.admitOrder = admitOrder;
    }

    public Integer getFollowClass() {
        return followClass;
    }

    public void setFollowClass(Integer followClass) {
        this.followClass = followClass;
    }

    public Integer getAccommodation() {
        return accommodation;
    }

    public void setAccommodation(Integer accommodation) {
        this.accommodation = accommodation;
    }

    public Integer getStudyMode() {
        return studyMode;
    }

    public void setStudyMode(Integer studyMode) {
        this.studyMode = studyMode;
    }

    public Integer getMigrantWorkersChildren() {
        return migrantWorkersChildren;
    }

    public void setMigrantWorkersChildren(Integer migrantWorkersChildren) {
        this.migrantWorkersChildren = migrantWorkersChildren;
    }

    public Integer getCountryWorkersChildren() {
        return countryWorkersChildren;
    }

    public void setCountryWorkersChildren(Integer countryWorkersChildren) {
        this.countryWorkersChildren = countryWorkersChildren;
    }

    public Integer getGovernmentPurchaseDegree() {
        return governmentPurchaseDegree;
    }

    public void setGovernmentPurchaseDegree(Integer governmentPurchaseDegree) {
        this.governmentPurchaseDegree = governmentPurchaseDegree;
    }

    public String getEnrollmentstatus() {
        return enrollmentstatus;
    }

    public void setEnrollmentstatus(String enrollmentstatus) {
        this.enrollmentstatus = enrollmentstatus;
    }

    public Integer getPoliticalStatus() {
        return politicalStatus;
    }

    public void setPoliticalStatus(Integer politicalStatus) {
        this.politicalStatus = politicalStatus;
    }

    public String getHobby() {
        return hobby;
    }

    public void setHobby(String hobby) {
        this.hobby = hobby;
    }

    public String getPrincipalSheet() {
        return principalSheet;
    }

    public void setPrincipalSheet(String principalSheet) {
        this.principalSheet = principalSheet;
    }

    public Integer getHealthCondition() {
        return healthCondition;
    }

    public void setHealthCondition(Integer healthCondition) {
        this.healthCondition = healthCondition;
    }

    public Integer getLeftBehindChildren() {
        return leftBehindChildren;
    }

    public void setLeftBehindChildren(Integer leftBehindChildren) {
        this.leftBehindChildren = leftBehindChildren;
    }

    public Integer getDisableType() {
        return disableType;
    }

    public void setDisableType(Integer disableType) {
        this.disableType = disableType;
    }

    public Integer getOrphan() {
        return orphan;
    }

    public void setOrphan(Integer orphan) {
        this.orphan = orphan;
    }

    public Integer getRuralDibao() {
        return ruralDibao;
    }

    public void setRuralDibao(Integer ruralDibao) {
        this.ruralDibao = ruralDibao;
    }

    public Integer getExtremelyPoor() {
        return extremelyPoor;
    }

    public void setExtremelyPoor(Integer extremelyPoor) {
        this.extremelyPoor = extremelyPoor;
    }

    public Integer getSingleParent() {
        return singleParent;
    }

    public void setSingleParent(Integer singleParent) {
        this.singleParent = singleParent;
    }

    public Integer getSolderChildren() {
        return solderChildren;
    }

    public void setSolderChildren(Integer solderChildren) {
        this.solderChildren = solderChildren;
    }

    public Integer getCausesOfPoverty() {
        return causesOfPoverty;
    }

    public void setCausesOfPoverty(Integer causesOfPoverty) {
        this.causesOfPoverty = causesOfPoverty;
    }

    public Integer getApplySponsor() {
        return applySponsor;
    }

    public void setApplySponsor(Integer applySponsor) {
        this.applySponsor = applySponsor;
    }

    public Integer getNeedSupply() {
        return needSupply;
    }

    public void setNeedSupply(Integer needSupply) {
        this.needSupply = needSupply;
    }

    public Integer getOnlyChild() {
        return onlyChild;
    }

    public void setOnlyChild(Integer onlyChild) {
        this.onlyChild = onlyChild;
    }

    public Integer getMartyrChildren() {
        return martyrChildren;
    }

    public void setMartyrChildren(Integer martyrChildren) {
        this.martyrChildren = martyrChildren;
    }

    public Integer getHealthfulDiet() {
        return healthfulDiet;
    }

    public void setHealthfulDiet(Integer healthfulDiet) {
        this.healthfulDiet = healthfulDiet;
    }

    public Integer getPutOnRecord() {
        return putOnRecord;
    }

    public void setPutOnRecord(Integer putOnRecord) {
        this.putOnRecord = putOnRecord;
    }

    public Integer getDeleteMember() {
        return deleteMember;
    }

    public void setDeleteMember(Integer deleteMember) {
        this.deleteMember = deleteMember;
    }

    public String getOrdernumber() {
        return ordernumber;
    }

    public void setOrdernumber(String ordernumber) {
        this.ordernumber = ordernumber;
    }

    @Override
    public String toString() {
        return "StudentTotal{" +
        "stuName=" + stuName +
        ", pstuName=" + pstuName +
        ", photoSrc=" + photoSrc +
        ", stuNumber=" + stuNumber +
        ", cardNumber=" + cardNumber +
        ", idCard=" + idCard +
        ", phoneNumber=" + phoneNumber +
        ", birth=" + birth +
        ", gender=" + gender +
        ", nation=" + nation +
        ", bloodType=" + bloodType +
        ", householdRegisterType=" + householdRegisterType +
        ", youthLeagueDate=" + youthLeagueDate +
        ", youthLeagueNum=" + youthLeagueNum +
        ", residenceAddr=" + residenceAddr +
        ", currentAddr=" + currentAddr +
        ", householdRegisterFront=" + householdRegisterFront +
        ", householdRegisterMain=" + householdRegisterMain +
        ", householdRegisterSelf=" + householdRegisterSelf +
        ", stuType=" + stuType +
        ", stuClass=" + stuClass +
        ", schoolNum=" + schoolNum +
        ", totalScore=" + totalScore +
        ", testScore=" + testScore +
        ", graduateSchool=" + graduateSchool +
        ", registerType=" + registerType +
        ", studyType=" + studyType +
        ", admitOrder=" + admitOrder +
        ", followClass=" + followClass +
        ", accommodation=" + accommodation +
        ", studyMode=" + studyMode +
        ", migrantWorkersChildren=" + migrantWorkersChildren +
        ", countryWorkersChildren=" + countryWorkersChildren +
        ", governmentPurchaseDegree=" + governmentPurchaseDegree +
        ", enrollmentstatus=" + enrollmentstatus +
        ", politicalStatus=" + politicalStatus +
        ", hobby=" + hobby +
        ", principalSheet=" + principalSheet +
        ", healthCondition=" + healthCondition +
        ", leftBehindChildren=" + leftBehindChildren +
        ", disableType=" + disableType +
        ", orphan=" + orphan +
        ", ruralDibao=" + ruralDibao +
        ", extremelyPoor=" + extremelyPoor +
        ", singleParent=" + singleParent +
        ", solderChildren=" + solderChildren +
        ", causesOfPoverty=" + causesOfPoverty +
        ", applySponsor=" + applySponsor +
        ", needSupply=" + needSupply +
        ", onlyChild=" + onlyChild +
        ", martyrChildren=" + martyrChildren +
        ", healthfulDiet=" + healthfulDiet +
        ", putOnRecord=" + putOnRecord +
        ", deleteMember=" + deleteMember +
        ", ordernumber=" + ordernumber +
        "}";
    }
}
